lua/cokeline/buffers.lua

```diff
@@ -339,6 +341,8 @@ local get_visible_buffers = function()
 
   for i, buffer in ipairs(_G.cokeline.visible_buffers) do
     buffer.index = i
+    buffer.is_first = i == 1
+    buffer.is_last = i == #_G.cokeline.visible_buffers
```

Comment:
There's no need to set these in every iteration of the loop, just set them once before the loop
```lua
_G.cokeline.visible_buffers[1].is_first = true
_G.cokeline.visible_buffers[#_G.cokeline.visible_buffers].is_first = true
for ...
```